src/browser/api/system.js

```diff
@@ -39,7 +39,23 @@ import ofEvents from '../of_events';
 const ProcessTracker = require('../process_tracker.js');
 import route from '../../common/route';
 import { fetchAndLoadPreloadScripts } from '../preload_scripts';
-
+const verbose = 'verbose';
+const info = 'info';
+const warning = 'warning';
+const error = 'error';
+const fatal = 'fatal';
+const logLevelMappings = new Map([
```

Comment:
Because this becomes a native call, I juts mirrored the levels as that is what you have to pass anyway, using the array index would be a bit clunky with the `index - 1` what would have to be used